packages/geo/src/lib/layer/shared/layers/vector-layer.ts

```diff
@@ -452,12 +460,29 @@ export class VectorLayer extends Layer {
         ? new olProjection({ code: paramsWFS.srsName })
         : proj;
       const currentExtent = olproj.transformExtent(extent, proj, wfsProj);
+      const ogcFilters = (options as OgcFilterableDataSourceOptions).ogcFilters;
+
+      if (
+        (this.previousLoadExtent &&
+          this.previousLoadExtent !== currentExtent) ||
+        (this.previousLoadResolution &&
+          this.previousLoadResolution !== resolution) ||
+        (this.previousOgcFilters && this.previousOgcFilters !== ogcFilters)
+      ) {
+        vectorSource.removeLoadedExtent(this.previousLoadExtent);
+        this.xhrAccumulator.map((xhr) => xhr.abort());
```

Comment:
On pourrait utiliser le forEach au lieu du map, il est plus performant pour la simple itération et encore mieux serait le "for of".